### PR TITLE
Add login button on the top of the landing page

### LINKED ISSUE #61: Add "log in" button to the login page
After implementing the new design for login page, we don't have a "log in" button anymore.

We have a "sign up" button in the very bottom of the page, but we should also have a "sign in" button at the top of the page for the users who already have an account and just want to login. Scrolling through the whole presentation to the bottom of the page, each time you want to login, is not handy.

So add a "log in" / "sign in" button at the top of the login page.

### PR DESCRIPTION
**Pull request by TomaTomoe**
- Duplicate GoogleSignUp button styled-component

- Move out common styles into Button styled-component

- Reuse Button styled-component in GoogleSignUp and GoogleLoginButton

Implement #61

### CODE REVIEW

File: src/components/LoginPage/LoginPageComponent.js [hunk 1]
```diff
@@ -30,11 +30,37 @@ const H3 = styled.h3`
...
+  padding: 2vmin 4vmin 2vmin calc(9vmin + 2%);
+
+  img {
+    height: 92%;
+    background-color: ${COLORS.white};
+    position: absolute;
+    padding: 3%;
+    left: 0.75%;
+    top: 4%;
+    border-radius: 0.4vmin;
```
Comment (DarthVanger) [inline src/components/LoginPage/LoginPageComponent.js:49]: The styles were copy-pasted from lines 72-79

copypaste

Could you please make those styles reusable instead? :)
I.e. to have them coded only once and used for both buttons.

**Comment by DarthVanger** [comment]
Please your commit message
```
Add login button on the top of the landing page. Issue 61
```
to
```
Add login button on the top of the landing page

Implement #61
```

Why?
1. "Issue 61" is NOT getting linked to the github issue #61.
    It should start with hash # to get linked to the issue
2. It's not the main info, we don't need it in the commit title. We can move it to the commit body.

File: src/components/LoginPage/LoginPageComponent.js [hunk 2]
```diff
@@ -30,14 +30,10 @@ const H3 = styled.h3`
   color: ${COLORS.white};
 `;
 
-// 2% - indent for image positioning
-// 92% - image height without vertical indention
-// 4% - vertical indention
-// 0.75% - horizontal indention
-const GoogleLoginButton = styled.button`
+const buttonIn = css`
```
Comment (DarthVanger) [inline src/components/LoginPage/LoginPageComponent.js:33]: What does `buttonIn` mean? :)

Button in... what? :)

File: src/components/LoginPage/LoginPageComponent.js [hunk 3]
```diff
@@ -30,14 +30,10 @@ const H3 = styled.h3`
...
   color: ${COLORS.white};
   background-color: ${COLORS.blue};
-  margin: auto;
+  float: right;
```
Comment (DarthVanger) [inline src/components/LoginPage/LoginPageComponent.js:36]: why changing float?
Won't the sign up button at the bottom the page be affected by this change? I.e. Move to the right from center?

File: src/components/LoginPage/LoginPageComponent.js [hunk 4]
```diff
@@ -60,6 +56,18 @@ const GoogleLoginButton = styled.button`
...
 `;
 
+const GoogleLoginButton = styled.button`
+  ${buttonIn}
+`;
+
+// 2% - indent for image positioning
+// 92% - image height without vertical indention
+// 4% - vertical indention
+// 0.75% - horizontal indention
```
Comment (DarthVanger) [inline src/components/LoginPage/LoginPageComponent.js:66]: Why do we have this comment here, what does it mean?

File: src/components/LoginPage/LoginPageComponent.js (hunk 2 above)
Comment (TomaTomoe) [reply src/components/LoginPage/LoginPageComponent.js:33]: Yeah, not the best of my decisions )  What about named it just Button? It is the common button style for this page

**Comment by DarthVanger** [comment]
Please squash commits

**Pull request merged**